protocol/messenger_pairing_and_syncing.go

```diff
@@ -512,6 +520,23 @@ func (m *Messenger) EnableInstallation(id string) error {
 	return nil
 }
 
+// EnableInstallationV2 enables an installation and returns the installation
+func (m *Messenger) EnableInstallationV2(id string) (*multidevice.Installation, error) {
```

Comment:
The V2 function seems to be identical, except for an extra return value. If that's correct, then I don't see the need to create a new function and deprecate the old one. We can just change the original function to return an extra value and change all callers that don't care about the `installation` to ignore with an underscore. This refactor would be guaranteed by the compiler, so no risk of regression, and we wouldn't create unnecessary tech debt.

Reply:
fixed in latest commit